test/Serilog.Tests/Serilog.Tests.csproj

```diff
@@ -21,6 +21,7 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" PrivateAssets="all" />
     <PackageReference Include="xunit" Version="2.4.2" />
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
+    <PackageReference Include="Shouldly" Version="4.1.0" />
```

Comment:
Keen not to introduce a new inconsistent set of assertions into this project.

Reply:
done